simulariumio/data_objects/agent_data.py

```diff
@@ -95,3 +99,165 @@ def __init__(
         self.n_subpoints = n_subpoints
         self.subpoints = subpoints
         self.draw_fiber_points = draw_fiber_points
+        self.type_ids = type_ids
+
+    @staticmethod
+    def _get_buffer_data_dimensions(buffer_data: Dict[str, Any]) -> Tuple[int]:
+        """"""
+        bundleData = buffer_data["spatialData"]["bundleData"]
```

Comment:
i just noticed this bundleData should really be bundle_data to conform to Python variable naming conventions. I believe either `black` or `flake8` should catch these things and let you know about them

Reply:
I always run black and I think flake8 is run as part of the build, but they don't catch this

Reply:
I noticed that too, the build runs both.  I could have sworn that in my editor i see those things reported but maybe they are different config settings.